routes/series.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const router = express.Router()
+const fetch = require('node-fetch')
+require('dotenv').config()
+
+const { API_BASE_URL, API_KEY, API_LANGUAGE } = process.env
+
+router.get('/:id', async (req, res) => {
+  const { id } = req.params
+  const url = `${API_BASE_URL}/${id}?api_key=${API_KEY}&language=${API_LANGUAGE}`
+
+  await fetch(url)
+    .then(res => res.json())
+    .then(data => res.send(data))
```

Comment:
You usually use async/await to get rid of `then` like that:
```suggestion
  // fetchResponse instead of `res` because you already use that name
  const fetchResponse = await fetch(url) 
  const data = await fetchResponse.json()
  res.send(data)
```